test/message-based/PriceAware.streamr-storage.test.ts

```diff
@@ -75,4 +75,54 @@ describe("Price Aware - streamr", function () {
       await syncTime(); // recommended for hardhat test
       await sample.executeWithPrice(toBytes32("AVAX"));
   });
+
+  it("should get price with single asset from streamr (first-valid)", async function () {
```

Comment:
Why do we only test a single dataSource (streamr-storage). 
Shouldn't we try to test at least 2 of them to check if it ends delivering the first valid price?

Reply:
We test few of them in other tests, where we use default data Source config for redstone-avalanche price feed. It uses many sources. I've also added similar tests to `PriceAwareWithManySigners.test.ts`